python/vineyard/deploy/local.py

```diff
@@ -148,4 +153,67 @@ def start_vineyardd(etcd_endpoints=None,
             etcd_ctx.__exit__(None, None, None)  # pylint: disable=no-member
 
 
+def init(socket=None, etcd_prefix=None, **kw):
+    '''
+    help new users to launch a local vineyardd and get a client as easy as possible
+
+    In a clean enviroment, simply use:
+    
+    .. code::
+    
+        vineyard.init()
+    
+    it will launch a local vineyardd and return a connected client to the vineyardd.
+
+    If the user has already lauched a vineyardd with a **SOCKET**. She can either set VINEYARD_IPC_SOCKET
+    as the SOCKET, and then run vineyard.init(), or simply use:
+    
+    .. code::
+    
+        vineyard.init(socket=SOCKET)
```

Comment:
Why these part? As we have discussed, DON'T make things so complicate....

Reply:
As we have discussed, `init` will always do launching, and `shutdown` will always do cleanup.